src/components/Footer/WebsiteFooter.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Footer } from './Footer';
+import { Text } from '../Text/Text';
+import styled from 'styled-components';
+import Socials from '../../Assets/dark/Socials.png';
+
+export const MiddleFooterContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const IconContainer = styled.img`
+  max-width: 100px;
+  height: auto;
+`;
+
+export const RightFooterContainer = styled.div`
```

Comment:
`MiddleFooterContainer` and `RightFooterContainer` have the same styling so you could use one styled footer container to avoid duplication

Reply:
Good point!